<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCTOCRDialog</class>
 <widget class="QDialog" name="TCTOCRDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCT Image OCR</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="imagePreview">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Image Preview</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="uploadButton">
       <property name="text">
        <string>Upload Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteButton">
       <property name="text">
        <string>Paste from Clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="doneButton">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui> 